### PR TITLE
[docs] python snippets for migration pages

### PR DESCRIPTION
**Pull request by AlexeyLebedev1**
### Tickets:
 - *82617*

### CODE REVIEW

File: docs/snippets/ie_common.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+def process_results(results, frame_id):
+    pass
+
+#! [ie:start_async_and_wait]
+# Start async inference on a single infer request
+infer_request.async_infer()
+# Wait for 1 milisecond
+infer_request.wait(1)
+# Wait for inference completion
+infer_request.wait()
```
Comment (jiwaszki) [inline docs/snippets/ie_common.py:50]: Isn't old API returning StatusCode from waits? Maybe it is worth mentioning.

**Review by jiwaszki — COMMENTED** [review]
Overall it is looking good

**Comment by ilya-lavrenov** [comment]
@jiwaszki do you approve the changes? We want to merge them to update documentation for 2022.1

**Comment by jiwaszki** [comment]
@ilya-lavrenov As I mentioned "Overall snippet code is looking good" - only thing to improve is mentioning of StatusCode returns in old API. But this is only an improvement:) We can merge it in current state.

**Review by ilya-lavrenov — APPROVED** [review]
@jiwaszki I've asked because I want explicit approve from you :)

**Pull request merged**